transaction/extra/src/tx_summary_unblinding/verifier.rs

```diff
@@ -313,7 +313,14 @@ impl TxSummaryStreamingVerifier {
         mut self,
         fee: Amount,
         tombstone_block: u64,
-    ) -> ([u8; 32], TxSummaryUnblindingReport) {
+    ) -> Result<([u8; 32], TxSummaryUnblindingReport), Error> {
+        if self.output_count != self.expected_num_outputs {
+            return Err(Error::StillExpectingMoreOutputs);
+        }
+        if self.input_count != self.expected_num_inputs {
+            return Err(Error::StillExpectingMoreInputs);
+        }
```

Comment:
It's kind of weird that this change is in the diff

Reply:
Wait what?

Reply:
I guess I needed a rebase.  Its gone now.